lib/video_controller.dart

```diff
@@ -6,7 +6,7 @@ class VideoController {
   Future<void> recordVideo(String filePath, Orientation? orientation) async {
     // We need to refresh camera before using it
     // audio channel need to be ready
-    CamerawesomePlugin.refresh();
+    await CamerawesomePlugin.refresh();
```

Comment:
I tested the code on Android without this line and there it seems to work. It might be worth to check it on iOS and remove it if it still works as expected. I'm also fine to check this in a later stage.

Reply:
It also seemed to work on iOS, but I'm unsure of the consequences when e.g. enabling/disabling mic permission, or when taking a picture before/after recording a video. Same goes for testing older Android/iOS versions. I think we should test this more before actually removing this line from the lib.